source/administration/config-settings.rst

```diff
@@ -3462,9 +3462,9 @@ Group Unread Channels (Experimental)
 
 **Default Off**: Disables the unread channels sidebar section by default. Users can turned it on in **Account Settings** > **Sidebar**.
 
-+------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature’s ``config.json`` setting is ``"ExperimentalGroupUnreadChannels": true`` as ``default_on`` or `"ExperimentalGroupUnreadChannels": false` as ``disabled``. |
-+------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
The setting doesn't show up for me in 'preview' mode. Can you help check that?

Reply:
I added a comment in the description that the formatting is not working. Not sure what to do.